backend/main.py

```
@@ -0,0 +1,227 @@
+# backend/main.py
+
+from fastapi import FastAPI
+from fastapi.middleware.cors import CORSMiddleware
+import requests
+import random
+import passiogo
+import numpy as np
```

Ruff (0.12.2)

`numpy` imported but unused

Remove unused import: `numpy`

(F401)